bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Widget.java

```diff
@@ -64,7 +64,7 @@ public abstract class Widget {
 	 *
 	 * @noreference This field is not intended to be referenced by clients.
 	 */
-	protected int nativeZoom;
+	public int nativeZoom;
```

Comment:
Why is this necessary?

Reply:
Because we need the nativeZoom to be used somewhere outside the package.

Reply:
"somwhere"?

Reply:
just saw in SWT_AWT as well

Reply:
I see. Thank you for the hints! Then it seems to be unavoidable to make it public.